db/migrate/20160924213020_change_katello_widget_names.rb

```diff
@@ -0,0 +1,23 @@
+class ChangeKatelloWidgetNames < ActiveRecord::Migration
+  def up
+    Widget.where(:name => 'Errata Widget').update_all(:name => 'Latest Errata')
+    Widget.where(:name => 'Content Views Widget').update_all(:name => 'Content Views')
+    Widget.where(:name => 'Content Host Subscription Status Widget').\
```

Comment:
The trailing slash should not be needed here or anywhere else below where its used